File: Back-end/app/Models/EmergencyContacts.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class EmergencyContacts extends Model
{
    use HasFactory;
    protected $fillable = [
        'user_id',
        'full_name',
        'phone_number',
        'relationship_status',
        'priority_number',
    ];

    // Define the relationship with the User model
    public function user()
    {
        return $this->belongsTo(User::class);
    }
}

File: Front-end/lib/DB/EmergencyContacts.dart
class EmergencyContacts{
  final String fullName;
  final String phoneNumber;
  final String relationshipStatus;

  EmergencyContacts({
    required this.fullName,
    required this.phoneNumber,
    required this.relationshipStatus,
  });

  factory EmergencyContacts.fromJson(jsonData){
    return EmergencyContacts(
      fullName: jsonData['full_name'],
      phoneNumber: jsonData['phone_number'],
      relationshipStatus: jsonData['relationship_status'],
    );
  }
}